crates/resolver/src/config.rs

```diff
@@ -889,6 +891,7 @@ impl Default for ResolverOpts {
             server_ordering_strategy: ServerOrderingStrategy::default(),
             recursion_desired: true,
             authentic_data: false,
+            shuffle_dns_servers: true,
```

Comment:
Can we flip this to default to false? I think that would be preferable for folks so that the library doesn't dramatically change from underneath them.

Reply:
Done :)